src/views/LocusGeneDiseaseView.vue

```diff
@@ -89,17 +85,14 @@ export default {
               <h4>Cross Cutting Modifier</h4>
             </td>
             <td class="w-75">
-              <p
-                v-if="
-                  locusGeneDiseaseData.cross_cutting_modifier &&
-                  locusGeneDiseaseData.cross_cutting_modifier.length > 0
-                "
-              >
+              <p v-if="locusGeneDiseaseData.cross_cutting_modifier &&
+      locusGeneDiseaseData.cross_cutting_modifier.length > 0
+      ">
```

Comment:
The code across the file is not formatted correctly, please have a look and fix it